manifests/oioswift.pp

```diff
@@ -48,6 +48,7 @@
   $middleware_ratelimit          = undef,
   $middleware_gatekeeper         = {},
   $middleware_healthcheck        = {},
+  $middleware_versionning        = true,
```

Comment:
Should be `$middleware_versioning` with only one 'n'.

Reply:
done